src/components/booklist.jsx

```diff
@@ -24,7 +25,11 @@ const BookList = () => {
                 <div className="vertical" />
                 <li>
                   &emsp;
-                  <button className="btn btn-outline-light" type="button" onClick={() => handleRemoveBook(book.item_id)}>
+                  <button
+                    className="btn btn-outline-light"
+                    type="button"
+                    onClick={() => handleRemoveBook(book.id)}
+                  >
                     Remove
                   </button>
```

Comment:
- Kindly make sure your remove button sticks to the Zeplin design:

Your button:
![Screenshot from 2023-04-15 23-50-29](https://user-images.githubusercontent.com/97834160/232254527-d0571aa3-9474-436c-94d5-a730b4a0076b.png)


Zeplin:

![Screenshot from 2023-04-15 23-51-29](https://user-images.githubusercontent.com/97834160/232254647-2ec03c46-a4d8-4408-ad32-c4f1489981d7.png)